safedelete/tests/test_single_delete_call_on_inheritance.py

```diff
@@ -0,0 +1,49 @@
+from ..config import (HARD_DELETE, SOFT_DELETE, SOFT_DELETE_CASCADE, HARD_DELETE_NOCASCADE, NO_DELETE)
+from ..models import SafeDeleteModel
+from django.test import TestCase
+
+
+class TestDeleteModel(SafeDeleteModel):
+    def __init__(self, *args, **kwargs):
+        # Initialize a counter to count the number of times delete method is called.
+        self.delete_call_counter = 0
+        super().__init__(*args, **kwargs)
```

Comment:
To support Python 2.7, you need to add the arguments to `super()` here.

Reply:
It's all right, we don't support python 2 anymore (only Django >= 2.2).